bciers/libs/components/src/form/fields/SourceTypeBoxTemplate.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+import { FieldTemplateProps } from "@rjsf/utils";
+import {
+  Grid,
+  Card,
+  CardActions,
+  CardHeader,
+  Collapse,
+  IconButton,
+  Paper,
+} from "@mui/material";
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
+import ArrowDropUpIcon from "@mui/icons-material/ArrowDropUp";
+import { useState } from "react";
+
+function SourceTypeBoxTemplate({
+  classNames,
+  label,
+  help,
+  description,
+  errors,
+  children,
+}: FieldTemplateProps) {
+  const [expand, setExpand] = useState(true);
+  return (
+    <>
+      <Paper className={classNames} sx={{ marginBottom: "10px" }}>
+        <Card style={{ textAlign: "left" }}>
+          <Grid container spacing={1}>
+            <Grid item xs={10}>
+              <CardHeader
+                sx={{ color: "blue" }}
+                titleTypographyProps={{ variant: "h6", color: "#38598A" }}
+                title={label}
+              />
+            </Grid>
+            <Grid item xs={1}>
+              <CardActions>
+                <IconButton onClick={() => setExpand(!expand)}>
+                  {expand ? <ArrowDropUpIcon /> : <ArrowDropDownIcon />}
+                </IconButton>
+              </CardActions>
+            </Grid>
+          </Grid>
+          <Collapse
+            in={expand}
+            sx={{
+              marginLeft: "30px",
+              marginRight: "30px",
+              marginTop: "10px",
+              marginBottom: "10px",
+            }}
+          >
+            {/* <Typography sx={{ paddingLeft: '30px', paddingRight: "30px", paddingTop: "10px"}}> */}
```

Comment:
Is this commented code still needed for the future?

Reply:
nope!